tcms/templates/email/post_bug_update/notify_new_assignee.txt

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+{% blocktrans%}Dear {{ assignee }},
+You've been assigned a new bug by {{ reporter }}.
+
+Please find the details below:
+
+{{ summary }}
```

Comment:
Can we use `{{ bug.summary }}` here instead of passing every variable independently? If yes that would simplify the number of context arguments passed when calling this template from the emailing function above.

Reply:
Actually see `email/post_run_save/email.txt` which passes only 1 argument as context variable but then blocktrans makes the deconstruction into several other context variables. This gives you the flexibility to easily render the email body as expected text in the test suite.